docs/source/set_dev_env_with_vagrant.rst

```diff
@@ -8,15 +8,31 @@ development environment, all you need is just to run
 
     vagrant up --provider virtualbox
 
+Note if you're under proxy, then you need to modify Vagrantfile first by adding proxy
+
+::
+
+    config.proxy.http = "http://;proxy>:<port>"
+    config.proxy.https = "http://<proxy>:<port>"
+
 After ``vagrant`` succeeds to run the virtual machine, you will get a complete
 environment to develop Nitrate,
 
 * a Python virtual environment creatd at ``$HOME/nitrate-env/`` with all
   necessary dependecies installed.
+ 
+* install all required Python dependencies
+
+  ::
+
+    cd /code
+    sudo pip install -r requirements/base.txt
+    sudo pip install -r requirements/devel.txt
 
 * database is created in MariaDB and name is ``nitrate``. It's empty. Before
-  hacking and running development server, remmeber to synchronize database
-  from models.
+  hacking and running development server, remember to synchronize database
+  from models. 
+  MySQL user is 'nitrate' without a password.  
```

Comment:
If you read the `tcms/settings/devel.py`, you should know that Nitrate uses default `root` username to log in MariaDB instance without a password. So, this line is incorrect.

Reply:
At the time when I was playing with it, it was exactly `nitrate`